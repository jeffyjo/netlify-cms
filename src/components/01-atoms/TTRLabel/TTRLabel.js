import React from 'react'
import PropTypes from 'prop-types'

import './TTRLabel.scss'

const TTRLabel = ({ TTRText }) => {
  return (
    <div className="a-ttr-label">Time to Read: {TTRText} min.</div>
  )
}

TTRLabel.proptypes = {
  TTRText: PropTypes.string.isRequired
}

export default TTRLabel