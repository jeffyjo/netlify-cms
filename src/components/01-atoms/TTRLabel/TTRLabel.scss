.a-ttr-label {
  font-size: .8em;
  font-weight: 600;
}