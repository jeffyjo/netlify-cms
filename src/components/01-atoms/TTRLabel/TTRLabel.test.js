import React from 'react'
import TTRLabel from './TTRLabel'
import renderer from 'react-test-renderer'

describe('Atom | TTRLabel', () => {
  let component = null
  let tree = null

  beforeEach(() => {
    component = renderer.create(
      <TTRLabel TTRText="10" />
    )
    tree = component.toJSON()
  })

  test('TTRLabel should render', () => {
    expect(tree).toMatchSnapshot()
  })

  test('TTRLabel should have text', () => {
    expect(tree.children[1]).toEqual('10')
  })

  test('TTRLabel should have default CSS class', () => {
    expect(tree.props.className).toEqual('a-ttr-label')
  })
})