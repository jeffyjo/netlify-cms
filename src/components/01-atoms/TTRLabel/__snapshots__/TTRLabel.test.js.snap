// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom | TTRLabel TTRLabel should render 1`] = `
<div
  className="a-ttr-label"
>
  Time to Read: 
  10
   min.
</div>
`;
